src/legacy/core_plugins/tile_map/public/tile_map_visualization.js

```diff
@@ -21,6 +21,7 @@ import { get } from 'lodash';
 import { GeohashLayer } from './geohash_layer';
 import { BaseMapsVisualizationProvider } from './base_maps_visualization';
 import { TileMapTooltipFormatterProvider } from './editors/_tooltip_formatter';
+import { getFormat } from '../../../ui/public/visualize/loader/pipeline_helpers/utilities';
```

Comment:
We are trying to remove all references to /ui/public folder. Maybe it make sence to pass this dependency from package.ts file ?